MapboxVisionAR/VIsionARManager/VisionARManagerDelegate.swift

```diff
@@ -16,10 +16,24 @@ public protocol VisionARManagerDelegate: AnyObject {
      Tells the delegate that AR lane was updated.
      */
     func visionARManager(_ visionARManager: VisionARManager, didUpdateARLane lane: ARLane?)
+
+    /**
+     Tells the delegate that AR mask was updated.
+     */
+    func visionARManager(_ visionARManager: VisionARManager, didUpdateARMaskImage: Image)
```

Comment:
Please mention the new methods in the changelog.

Reply:
done